notebooks/09_Monitoring/src/09_02_Iris_Pipeline_Metrics/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM tensorflow/tensorflow:1.15.2-py3
```

Comment:
Can you update path `09_02_Iris_Pipeline_Metrics` to something simple? User's entrypoint is notebook. Src folder is auxiliary resources. Make it something like `pipeline_ scalar_metrics` or `09_02`.

Reply:
Sure, let me fix it up.